site-packages/integralstor_utils/django_utils.py

```diff
@@ -0,0 +1,43 @@
+
+def in_request_parameters(request, param_name):
+    result = False
+    try:
+        if not request:
+            raise Exception('Request object not provided')
+        if not param_name:
+            raise Exception('Parameter name not provided')
+        if request.method == 'GET':
+            if param_name in request.GET:
+                result = True
+        elif request.method == 'POST':
+            if param_name in request.POST:
+                result = True
+        else:
+            raise Exception('Unsupported request method : %s' % request.method)
+    except Exception, e:
+        return None, 'Error checking request parameters : %s' % str(e)
+    else:
+        return result, None
+
+
+def get_request_parameter_value(request, param_name):
```

Comment:
Will we need at some point to get multiple parameter values? If so, then can we receive the argument `param_name` as a list and return a dict with values? Iterated call to this function passing single parameter as an argument may not be an efficient way to fetch multiple values.